gb-cpu/src/trait.rs

```diff
@@ -0,0 +1,9 @@
+use crate::address_bus::Area;
+
+trait Address {
+    /// Return the relative address in the current area
+    fn get_address() -> usize;
```

Comment:
Shouldn't we use u16 for adress ? Or do we use usize to avoid casting when using it as index in an array ?

Reply:
I return a `usize` to prevent the `user` of `Address` to cast to `usize` when indexing an array